.github/workflows/go_module.yml

```
@@ -28,12 +28,15 @@ jobs:
         run: if [ "$(gofmt -l . | wc -l)" -gt 0 ]; then exit 1; fi
 
       - name: "Lint"
-        # TODO: Expand to the other modules once proven / rectified
-        if: github.event.repository.name == 'go-wonde'
         id: lint
         uses: golangci/golangci-lint-action@v3
+        env:
+          GOPRIVATE: github.com/linc-technologies
         with:
+          # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: latest
+          # https://golangci-lint.run/usage/configuration/#command-line-options
+          args: --issues-exit-code=0 --build-tags=internal
```

Pins `golangci-lint` to use linters based on the version of Go we use :3

```suggestion
          args: --issues-exit-code=0 --build-tags=internal --go=${{ matrix.go }}
```